saltstack/base/salt/postgresql/init.sls

```diff
@@ -108,9 +108,9 @@ systemd-link:
     - repl: "[Install]\nAlias=postgresql.service"
     - unless: cat /usr/lib/systemd/system/postgresql-9.6.service | grep postgresql.service
 
-reenable-postgres:
+create-postgres-service-link:
```

Comment:
this will be effective for all images not just ycloud. Are you sure about this?

Reply:
Yes I know, I haven't tested those path, It might be safer to create a separate branch for it.
I'll use the "yarn-private directory check" to decide.